lib/Checkout/ApiClient.php

```diff
@@ -132,6 +133,10 @@ public function submitFile(string $path, FileRequest $fileRequest, SdkAuthorizat
                         "name" => "file",
                         "contents" => fopen($fileRequest->file, "r")
                     ],
+                    [
+                        "name" => "path",
```

Comment:
Didn't this affect the `submitFile` for disputes?
I was expecting only change the `name` property depends on if the submitFile is for internal or external, or maybe the configuration? With that we only send two properties either "name" => "file" or "name" => "path".

Reply:
It doesn't affect, but definitely this is not right!